cli/src/main/java/com/box/l10n/mojito/cli/command/param/Param.java

```diff
@@ -87,4 +87,8 @@
     public static final String EXPORT_LOCALES_SHORT = "-l";
     public static final String EXPORT_LOCALES_DESCRIPTION = "List of locales to be exported, format: fr-FR,ja-JP";
 
+    public static final String CHECK_SLA_LONG = "--check-sla";
+    public static final String CHECK_SLA_SHORT = "-cs";
+    public static final String CHECK_SLA_DESCRIPTION = "weather check sla or not for repository by default false";
```

Comment:
'weather check' --> 'wether to check'

don't need to specify default false if using a boolean with a default value. The doc will be auto generated

Reply:
change to --> "whether check sla or not for repository"